config/forms/evidence.yaml

```diff
@@ -37,22 +38,24 @@ lists:
         value: driving_licence_eea
       - text: Digital tachograph driver smart card
         value: driving_licence_tachograph
-      
+
       - text: Armed forces identity card
         value: id_card_forces
       - text: Biometric residence permit
         value: id_card_residence_permit
       - text: An identity card from an EU or European Economic Area (EEA) country that meets the Council Regulation (EC) No 2252/2004 standards
+        short_name: identity card
```

Comment:
Should be unique to every piece of evidence?

Reply:
The short name would be used on the overview to display the evidence, there should be a way to capitalise the first character

Reply:
I think I've now sorted the short name thing properly, and we have unique short names either explicitly set, or we default to using the text (with the appropriate case).